recipes/libfabric/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class LibfabricConan(ConanFile):
+    name = "libfabric"
+    description = "Open Fabric Interfaces"
+    topics = ("conan", "fabric")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://libfabric.org"
+    license = "BSD"
+    settings = "os", "arch", "compiler", "build_type"
+    _providers = ['gni', 'psm', 'psm2', 'psm3', 'rxm', 'sockets', 'tcp', 'udp', 'usnic', 'verbs', 'bgq']
+    options = {
+        **{ p: "ANY" for p in _providers },
```

Comment:
```suggestion
        **{ p: [True, False, "auto", "dl"] for p in _providers },
```

read comment below.
Also, if `auto` has same effect as True or False, it should have the same package id! For example: https://github.com/conan-io/conan-center-index/blob/41d4fbb8c218f9d938ccb8b9d888bff52002e62e/recipes/getdns/all/conanfile.py#L48

Reply:
I used `ANY` because it also be a directory path

Reply:
I can't know if `auto` will resolve to `True` of `False` before running configuration